src/dbnode/storage/types.go

```diff
@@ -866,21 +866,27 @@ type databaseShardRepairer interface {
 	) (repair.MetadataComparisonResult, error)
 }
 
-// databaseRepairer repairs in-memory database data.
-type databaseRepairer interface {
-	// Start starts the repair process.
-	Start()
+// BackgroundProcess is the interface for background processes that are being run by the database.
```

Comment:
nit; `BackgroundProcess is a background process that is run by the database.`

Reply:
ok